### PR TITLE
feat: rtl provider

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

To be RTL fully compatible our components needs to be aware if we are in rtl mode. We found out that a solution that uses context will be more scalable and keep our API cleaner

## 🚀 Changes

 
- Create DirectionProvider
- Create useDirection hook
- Add Storybook RTL addon

https://user-images.githubusercontent.com/16908937/112297490-de224f80-8c8d-11eb-888e-b2466f0978bc.mov

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/.storybook/main.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = {
     '@storybook/addon-a11y',
     '@storybook/addon-docs',
     '@storybook/addon-viewport',
+    '@pxblue/storybook-rtl-addon/register',
```
Comment (dozortsev) [inline packages/components/.storybook/main.js:18]: please show a screenshot of show this addon work and look

File: packages/components/src/common/hooks/useDirection/useDirection.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+import { DirectionContext } from '../../../provider/direction/DirectionProvider';
+
+export const useDirection = () => ({
+  direction: useContext(DirectionContext),
+  isRTL: useContext(DirectionContext) === 'rtl',
+});
```
Comment (dozortsev) [inline packages/components/src/common/hooks/useDirection/useDirection.js:7]:
```suggestion
export const useDirection = () => (
const context = useContext(DirectionContext);
return {
  direction: context,
  isRTL: context === 'rtl',
});
```

File: packages/components/src/common/hooks/useDirection/useDirection.spec.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+beforeEach(() => {
+  realUseContext = React.useContext;
+  React.useContext = jest.fn(() => direction);
+  useContextMock = React.useContext;
+});
+
+afterEach(() => {
+  React.useContext = realUseContext;
+});
+describe('useRTL', () => {
```
Comment (dozortsev) [inline packages/components/src/common/hooks/useDirection/useDirection.spec.js:18]: `useDirection`

**Review by dozortsev — COMMENTED** [review]
please remove those redundant (code formatting) changes from this PR

File: packages/components/.storybook/main.js (hunk 1 above)
Comment (andreapiras) [reply packages/components/.storybook/main.js:18]: Added in description

**Pull request merged**